OutOfSchool/OutOfSchool.WebApi/Models/ChildDTO.cs

```diff
@@ -29,7 +29,7 @@ public class ChildDto
     public DateTime DateOfBirth { get; set; } = default;
 
     [Required(ErrorMessage = "Gender is required")]
-    [Range(0, 1)]
+    [EnumDataType(typeof(Gender), ErrorMessage = "Gender should be in enum range")]
```

Comment:
There is a possibility to use a unified error message for all of the EnumDataType attributes. Please, create a string constant for the error message.
```suggestion
    [EnumDataType(typeof(Gender), ErrorMessage = "{0} should be in enum range")]
```